src/iexamine.cpp

```diff
@@ -752,33 +752,37 @@ void iexamine::rubble(player &p, const tripoint &examp)
 /**
  * Try to pry crate open with crowbar.
  */
-void iexamine::crate(player &p, const tripoint &examp)
+void iexamine::crate( player &p, const tripoint &examp )
 {
-    // Check for a crowbar in the inventory
-    bool has_prying_tool = p.crafting_inventory().has_quality( quality_id( "PRY" ), 1 );
+    // PRY 1 is sufficient for popping open a nailed-shut crate.
+    const bool has_prying_tool = p.crafting_inventory().has_quality( quality_id( "PRY" ), 1 );
+
     if( !has_prying_tool ) {
-        add_msg( m_info, _("If only you had a crowbar...") );
+        add_msg( m_info, _( "If only you had something to pry with..." ) );
         return;
     }
 
     // Ask if there's something possibly more interesting than this crate here
     // Shouldn't happen (what kind of creature lives in a crate?), but better safe than getting complaints
-    std::string xname = g->m.furnname(examp);
-    if( ( g->m.veh_at( examp ) ||
-          !g->m.tr_at( examp ).is_null() ||
-          g->critter_at( examp ) != nullptr ) &&
-          !query_yn(_("Pry that %s?"), xname.c_str() ) ) {
+    std::string xname = g->m.furnname( examp );
+    const auto is_vehicle = g->m.veh_at( examp );
+    const auto is_trap = !g->m.tr_at( examp ).is_null();
+    const auto is_critter = g->critter_at( examp ) != nullptr;
+    const auto query = query_yn( _( "Pry that %s?" ), xname );
```

Comment:
This changes the behavior, the previous code would only query if one or more of the previous tests passed.